src/drivers/cmakeDriver.ts

```diff
@@ -600,6 +631,48 @@ export abstract class CMakeDriver implements vscode.Disposable {
         log.debug(localize('cmakedriver.test.preset.set.to', 'CMakeDriver test preset set to {0}', testPreset?.name || null));
     }
 
+    /**
+     * Change the current package preset
+     * @param packagePreset The new package preset
+     */
+    async setPackagePreset(packagePreset: preset.PackagePreset | null): Promise<void> {
+        if (packagePreset) {
+            log.info(localize('switching.to.package.preset', 'Switching to package preset: {0}', packagePreset.name));
+        } else {
+            log.info(localize('unsetting.package.preset', 'Unsetting package preset'));
+        }
+
+        await this.doSetPackagePreset(async () => {
+            await this._setPackagePreset(packagePreset);
+        });
+    }
+
+    private async _setPackagePreset(packagePreset: preset.PackagePreset | null): Promise<void> {
```

Comment:
Nit: (NOT TO BE ADDRESSED IN THIS PR). 

I've noticed many uses of null vs undefined in this repo, and at some point we might should refactor to only using `undefined`